includes/admin/settings/class-settings-bootstrap.php

```diff
@@ -350,16 +350,18 @@ private static function settings_general() {
                 'group' => 'general/advanced',
             )
         );
-        wpbdp_register_setting(
-            array(
-                'id'      => 'enqueue-fontawesome-styles',
-                'type'    => 'checkbox',
-                'name'    => _x( 'Enqueue Business Directory\'s FontAwesome styles?', 'settings', 'business-directory-plugin' ),
-                'desc'    => _x( 'This helps to prevent conflicts with other plugins that already do this. Disable this only if you\'re having an issue with FontAwesome icons and have performed a conflict test to validate this is a multiple styles enqueueing issue.', 'settings', 'business-directory-plugin' ),
-                'default' => true,
-                'group'   => 'general/advanced',
-            )
-        );
+
+		wpbdp_register_setting(
+			array(
+				'id'      => 'enqueue-fontawesome-styles',
+				'type'    => 'hidden',
+				'class'   => 'hidden',
+				'name'    => _x( 'Enqueue Business Directory\'s FontAwesome styles?', 'settings', 'business-directory-plugin' ),
+				'desc'    => _x( 'This helps to prevent conflicts with other plugins that already do this. Disable this only if you\'re having an issue with FontAwesome icons and have performed a conflict test to validate this is a multiple styles enqueueing issue.', 'settings', 'business-directory-plugin' ),
```

Comment:
Are the name and description on the hidden field used anywhere now? Or can they be removed?